src/main/java/com/powsybl/openloadflow/OpenLoadFlowProvider.java

```diff
@@ -112,7 +112,7 @@ public static OpenLoadFlowParameters getParametersExt(LoadFlowParameters paramet
         return parametersExt;
     }
 
-    private static SlackBusSelector getSlackBusSelector(Network network, LoadFlowParameters parameters, OpenLoadFlowParameters parametersExt) {
+    public static SlackBusSelector getSlackBusSelector(Network network, LoadFlowParameters parameters, OpenLoadFlowParameters parametersExt) {
```

Comment:
Why do you need that change?

Reply:
I used it temporarily elsewhere and probably forgot to put it back to private when it was not needed any more. It can remain private.